vaadin-popover-flow-parent/vaadin-popover-flow/src/main/java/com/vaadin/flow/component/popover/PopoverVariant.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright 2000-2024 Vaadin Ltd.
+ *
+ * This program is available under Vaadin Commercial License and Service Terms.
+ *
+ * See <https://vaadin.com/commercial-license-and-service-terms> for the full
+ * license.
+ */
+package com.vaadin.flow.component.popover;
+
+import com.vaadin.flow.component.shared.ThemeVariant;
+
+/**
+ * Set of theme variants applicable for the {@code vaadin-popover} component.
+ */
+public enum PopoverVariant implements ThemeVariant {
+    ARROW("arrow");
+
+    private final String variant;
+
+    PopoverVariant(String variant) {
```

Comment:
Could we have a test?

Reply:
I think we don't have tests for other variant enums, not sure how to test this one.